### PR TITLE
Tctl admin db

### PR DESCRIPTION
**Pull request by DJSanti**
DB commands need to be reviewed

### CODE REVIEW

File: docs/tctl/admin/db/clean.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+title: tctl admin db clean
+description: Cleaning up corrupted Workflows
+tags:
+  - reference
+  - tctl
+  - admin
+  - db
+---
+
+The `tctl admin db clean` command cleans corrupted [Workflows](/docs/concepts/what-is-a-workflow) from the targeted database.
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:12]: Workflow Executions


And more specifically - Event Histories I think... can double check with @dtornow if we need to.

File: docs/tctl/admin/db/clean.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+#### `--db_engine value`
+
+Type of DB engine to use
+
+Default: `cassandra`
+Value: `cassandra` | `mysql` | `postgres`
+
+#### `--db_address value`
+
+Persistence address for the Database.
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:27]:
```suggestion
Persistence address for the database.
```

File: docs/tctl/admin/db/clean.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+#### `--db_port value`
+
+Persistence port for the DB.
+
+Default: 9042
+
+#### `--username value`
+
+DB username.
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:39]:
```suggestion
Database username.
```

File: docs/tctl/admin/db/clean.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+#### `--password value`
+
+DB password.
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:43]:
```suggestion
Database password.
```

File: docs/tctl/admin/db/clean.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+#### `--keyspace value`
+
+DB keyspace
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:47]:
```suggestion
Database keyspace
```

File: docs/tctl/admin/db/scan.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+title: tctl admin db scan
+description: Scanning for corrupted executions in a database
+tags:
+  - reference
+  - tctl
+  - admin
+  - db
+---
+
+The `tctl admin db scan` command scans concrete executions in a given database, and detects corrupted executions.
```
Comment (flossypurse) [inline docs/tctl/admin/db/scan.md:12]:
```suggestion
The `tctl admin db scan` command scans concrete Workflow Executions in a given database, and detects corrupted ones.
```
?

File: docs/tctl/admin/db/clean.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+---
+id: clean
+title: tctl admin db clean
+description: Cleaning up corrupted Workflows
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:4]:
```suggestion
description: How to clean up corrupted Workflows using tctl.
```

File: docs/tctl/admin/db/clean.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+tags:
+  - reference
+  - tctl
+  - admin
+  - db
+---
+
+The `tctl admin db clean` command cleans corrupted [Workflow Executions](/docs/concepts/what-is-a-workflow-execution) from the targeted database.
+
```
Comment (flossypurse) [inline docs/tctl/admin/db/clean.md:13]: What do you think about having some sort of step by step instructions - general usage steps kind of thing?

Comment (DJSanti) [reply docs/tctl/admin/db/clean.md:13]: I definitely don't mind. If anything, I'd love to put instructions for all these pages, but in later updates to them

File: docs/tctl/admin/index.md [hunk 9]
```diff
@@ -14,15 +14,15 @@ Alias : `adm`
...
-- tctl admin shard
-- tctl history_host
-- tctl taskqueue
-- tctl membership
-- tctl cluster
-- tctl dlq
-- tctl db
-- tctl decode
+- [`tctl admin workflow`](/docs/tctl/admin/workflow/index)
+- [`tctl admin shard`](/docs/tctl/admin/shard/index)
```
Comment (flossypurse) [inline docs/tctl/admin/index.md:18]:
```suggestion
- [`tctl admin workflow`](/docs/tctl/admin/workflow/)
- [`tctl admin shard`](/docs/tctl/admin/shard/)
```

If there is an index page there, then you can do it like this ^

**Review by flossypurse — APPROVED** [review]
Awesome, looking forward to seeing the next iteration.

**Pull request merged**